partner_external_map/__manifest__.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2015-2016 Akretion - Alexis de Lattre
+# Copyright 2016-2017 Tecnativa - Pedro M. Baeza
+# Copyright 2018 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Partner External Maps',
+    'version': '12.0.1.0.0',
+    'category': 'Extra Tools',
+    'license': 'AGPL-3',
+    'summary': 'Add Map and Map Routing buttons on partner form to '
+               'open GMaps, OSM, Bing and others',
+    'author': 'Akretion, '
+              'Tecnativa, '
+              'Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/partner-contact',
+    'depends': [
+        'base',
+    ],
+    'data': [
```

Comment:
@ernestotejeda Improve Data sequence should be [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L29)

Reply:
@nikul-serpentcs I wouldn't enforce that order as not always can be done that way due to XML-ID dependencies. There's no guideline about that either.

Reply:
Yes @pedrobaeza , But here `security` file is last, it's must be first

Reply:
@pedrobaeza Because `security` files  it's must be first not last

Reply:
No, where do you find that requirement?

Reply:
No, that's a template where the order was not intended. Only a accident.